restlight-core/src/main/java/io/esastack/restlight/core/resolver/rspentity/AbstractResponseEntityResolver.java

```diff
@@ -53,11 +53,11 @@ protected AbstractResponseEntityResolver(boolean maySimpleType) {
     }
 
     @Override
-    public HandledValue<Void> writeTo(ResponseEntity entity,
-                                      ResponseEntityChannel channel,
-                                      RequestContext context) throws Exception {
+    public Result<Void, Void> writeTo(ResponseEntity entity,
+                                ResponseEntityChannel channel,
+                                RequestContext context) throws Exception {
         if (!supports(entity)) {
-            return HandledValue.failed();
+            return Result.err();
```

Comment:
Err is a more general expression. In fact, if we really want to use a more accurate expression, failed is not very appropriate too. We should express it with unsupported. But for reuse, we reuse result.err(), instead of starting a new stove.in this case, there is no need to be too tangled about whether the expression is completely accurate, and the meaning is roughly the same.